packages/single-line/src/SingleLineEditor.tsx

```diff
@@ -56,7 +61,7 @@ export function SingleLineEditor(props: SingleLineEditorProps) {
               className={direction === 'rtl' ? styles.rightToLeft : ''}
               isRequired={field.required}
               isInvalid={errors.length > 0}
-              isDisabled={disabled}
+              isDisabled={props.isDisabled ?? disabled}
```

Comment:
How about we implement this in the [FieldConnector](https://github.com/contentful/field-editors/blob/master/packages/_shared/src/FieldConnector.ts) so we have a single place that computes whether a field is disabled or not? Currently, the FieldConnector already handles `isInitiallyDisabled` and manages the `disabled` attribute we get in the callback.

Reply:
The `isIntiallyDisabled` prop doesn't seem to be working properly. The editor is immediately reset to enabled from the context of app even if `isInitiallyDisabled` is set to `true`. Is this something Tolkien is aware of ?